tensorbay/client/job.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python3
+#
+# Copyright 2021 Graviti. Licensed under MIT License.
+#
+
+"""Basic structures of asynchronous jobs."""
+
+from typing import Any, Dict, Optional
+
+
+class Job:  # pylint: disable=too-many-instance-attributes
+    """This class defines :class:`Job`.
+
+    Arguments:
+        title: Title of the Job.
+        job_id: ID of the Job.
+        job_type: Type of the Job.
+        creator: The creator of the Job.
+        arguments: Arguments of the Job.
+        created_at: The time when the Job is created.
+        started_at: The time when the Job is started.
+        finished_at: The time when the Job is finished.
+        total_used_time: Time spent on the Job.
```

Comment:
`total_used_time` can be calculated from `finished_at - started_at`?
If so the `total_used_time ` is a redundant data, which can be removed.